frontend/src/components/TransactionCollapse.spec.js

```diff
@@ -2,8 +2,18 @@ import { mount } from '@vue/test-utils'
 import TransactionCollapse from './TransactionCollapse'
 import { GdtEntryType } from '../graphql/enums'
 
+import Vue from 'vue'
+
+// disable throwing Errors on warnings to catch the warning
+Vue.config.warnHandler = (w) => {}
+
 const localVue = global.localVue
 
+const consoleErrorMock = jest.fn()
+
+// eslint-disable-next-line no-console
+console.error = consoleErrorMock
```

Comment:
👀 mmm. Is this because of the `throw new Error...` or did you forget this when debugging?